src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,10 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+    private static final UserService userService = new UserService();
```

Comment:
Let's make it non-static, generally, it is easier to work with, and more OOP-like, which we will cover later on in the course :)